Source/WebCore/xml/DOMParser.h

```diff
@@ -30,10 +30,10 @@ class TrustedHTML;
 
 class DOMParser : public RefCounted<DOMParser> {
 public:
-    static Ref<DOMParser> create(Document& contextDocument);
-    ~DOMParser();
+    WEBCORE_EXPORT static Ref<DOMParser> create(Document& contextDocument);
+    WEBCORE_EXPORT ~DOMParser();
 
-    ExceptionOr<Ref<Document>> parseFromString(std::variant<RefPtr<TrustedHTML>, String>&&, const AtomString& contentType);
+    WEBCORE_EXPORT ExceptionOr<Ref<Document>> parseFromString(std::variant<RefPtr<TrustedHTML>, String>&&, const AtomString& contentType);
```

Comment:
Why is this being exported? The new caller seems to be in `Document`, which is in WebCore.

Reply:
ah these changes was used for testing; will revert.